src/FrontendAccountCreation.Web/Controllers/ReprocessorExporter/OrganisationController.cs

```diff
@@ -311,10 +312,50 @@ public async Task<IActionResult> IsOrganisationAPartner(IsOrganisationAPartnerVi
     [HttpGet]
     [Route(PagePath.RoleInOrganisation)]
     [OrganisationJourneyAccess(PagePath.RoleInOrganisation)]
```

Comment:
This will need the feature flag parameter adding, so that it isn't accessible if the feature flag isn't enabled.
(We'll probably refactor how the feature flag is handled, but for now, it needs it.)
```suggestion
    [OrganisationJourneyAccess(PagePath.RoleInOrganisation, FeatureFlags.AddOrganisationCompanyHouseDirectorJourney)]
```